Lib/test/test_json/test_tool.py

```diff
@@ -206,3 +207,14 @@ def test_ensure_ascii_default(self):
         # asserting an ascii encoded output file
         expected = [b'{', rb'    "key": "\ud83d\udca9"', b"}"]
         self.assertEqual(lines, expected)
+
+    @unittest.skipIf(sys.platform =="win32", "The test is failed with ValueError on Windows")
+    def test_broken_pipe_error(self):
+        cmd = [sys.executable, '-m', 'json.tool']
+        proc = subprocess.Popen(cmd,
+                                stdout=subprocess.PIPE,
+                                stdin=subprocess.PIPE)
+        proc.stdout.close()
+        proc.communicate(b'"a"')
```

Comment:
I would prefer valid JSON:
```suggestion
        proc.communicate(b'{}')
```